docs/index.md

```diff
@@ -31,20 +31,20 @@ Mocha is a feature-rich JavaScript test framework running on [Node.js][] and in
 - [reports test durations](#test-duration)
 - [highlights slow tests](#dot-matrix)
 - [file watcher support](#min)
-- [global variable leak detection](#-check-leaks)
-- [optionally run tests that match a regexp](#-grep-regexp-g-regexp)
-- [auto-exit to prevent "hanging" with an active loop](#-exit)
+- [global variable leak detection](#--check-leaks)
```

Comment:
this fixes the link in the Markdown file but breaks the link in the deployed HTML. The deployed HTML is much more important and we don't want to change the anchor as that would break other links to this section. Of course, we could add a redirect or a second anchor, but that's way out of scope I think.

For now, please revert changes to these links unless the fix HTML links. Again, let's disable any rule that's recommending we change these links: that rule isn't aware that this MD file is turning into an HTML one through a special process

Reply:
Applies throughout the file